src/cascadia/TerminalSettingsEditor/Actions.cpp

```diff
@@ -4,108 +4,14 @@
 #include "pch.h"

```

Comment:
Note to other reviewers: go to `...` --> "View File". This file is super short now and is much easier to review that way.